generate-stackbrew-library.sh

```diff
@@ -2,8 +2,8 @@
 set -eu
 
 declare -A aliases=(
-	[8.4]='8 latest'
-	[8.5-rc]='rc'
+	[8.4]='8'
+	[8.5-rc]='rc latest'
```

Comment:
Everything except this change seems pretty good to me -- I don't think it's reasonable to point `drupal:latest` to a pre-release.